content/3-lifecycle/2-on-unmount/lit/x-time.js

```diff
@@ -0,0 +1,26 @@
+import { LitElement, html } from 'lit';
+import { customElement, state } from 'lit/decorators.js';
+
+@customElement('x-time')
+export class XTime extends LitElement {
+	@state()
+	time = '';
+
+	timer;
+
+	connectedCallback() {
+		super.connectedCallback();
+		this.timer = setInterval(() => {
+			this.time = new Date().toLocaleTimeString();
+		}, 1000);
+	}
+
+	disconnectedCallback() {
+		super.disconnectedCallback();
+		clearInterval(this.timer);
+	}
```

Comment:
Perfect case for `connectedCallback` ! :)
https://lit.dev/docs/v1/components/events/#add-event-listeners-in-connectedcallback